inc/osvr/ClientKit/Imaging.h

```diff
@@ -94,29 +92,25 @@ namespace clientkit {
                                      const struct OSVR_ImagingReport *report) {
                 ImagingCallbackRegistration *self =
                     static_cast<ImagingCallbackRegistration *>(userdata);
-                ImagingReportOpenCV newReport;
+                ImagingReport newReport;
                 newReport.sensor = report->sensor;
                 newReport.buffer.reset(report->state.data,
                                        ImagingDeleter(self->m_ctx));
-                newReport.frame = cv::Mat(
```

Comment:
@rpavlik Just to quote the specific argument:

> What I really wouldn't want to see is the (slightly unsafe, less usable) C API being used (or a wrapper re-invented) in C++ games/engines because the wrapper uses things that aren't allowed.